### DISCUSSION TITLE
Export to JSON

### DISCUSSION THREAD

**Discussion by vanillajonathan** [NickvisionApps/Denaro#236] [Ideas]
With JSON you could easily serialize it into a class and run operations such as `filter`, `map`, `reduce`, etc.

This can be done with `System.Text.Json`.

**Comment by nlogozzo** [comment]
I don't see the purpose for this. What are you trying to do with `filter`, `map`, and `reduce`? 
Filtering, searching, and sorting are built into `Denaro` itself already. If there are certain filters or features that you'd like to see come from the application itself, feel free to open discussion requests for them :)

**Comment by nlogozzo** [comment]
As stated in #234, what's the point of exporting to a file format, editing everything from that file format, and then importing back to Denaro. Just use that file format and don't use Denaro.
If there are features missing from Denaro that lead you to use these file formats in the first place, we'd be more than happy to add them in.
